### PR TITLE
support all characters in cue voice span

### PR DESCRIPTION
**Pull request by finn0**
Hi, go-astisub provides great help for me, thank you for the hard work!
I made a PR so it can recognize non-ASCII characters in the cue voice span, please help to double check, thanks.

### CODE REVIEW

File: webvtt.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var (
 	bytesWebVTTItalicEndTag            = []byte("</i>")
 	bytesWebVTTItalicStartTag          = []byte("<i>")
 	bytesWebVTTTimeBoundariesSeparator = []byte(webvttTimeBoundariesSeparator)
-	webVTTRegexpStartTag               = regexp.MustCompile(`(<v([\.\w]*)([\s\w]+)+>)`)
+	webVTTRegexpStartTag               = regexp.MustCompile(`(<v([\.\w]*)(.{1,}?)>)`)
```
Comment (asticode) [inline webvtt.go:35]: Could you replace `{1,}` with `+` ?

**Review by asticode — CHANGES_REQUESTED** [review]
Nice PR, one tiny change needed though 👍

File: webvtt.go (hunk 1 above)
Comment (finn0) [reply webvtt.go]: Thank you for the suggestion. Pushed a new commit.

**Comment by asticode** [comment]
fyi I've created a `v0.23.0` tag

**Pull request merged**